lib/pavilion/plugins/sched/raw.py

```diff
@@ -284,3 +285,24 @@ def _cancel_job(self, test):
                 STATES.SCHED_ERROR,
                 "PID {} refused to die.".format(pid)
             )
+
+    def get_overall_status(self, test):
+
+        if not test.job_id:
+            return "Job ID not found."
+        else:
+            host, pid = test.job_id.rsplit('_', 1)
+
+        sched_stat = {}
+
+        regex = re.compile("(VmSize|Cpus_allowed_list):\s(.+)")
```

Comment:
Cpus_allowed_list isn't super useful. See `man proc` for info on other useful info you could add from other files. 
`/proc/[pid]/sched` might be useful too. I'm leaving it up to you to do the research and figure out a reasonable set of data to pull.

Reply:
Instead of `Cpus_allowed_list` I used `se.sum_exec_runtime` from the `sched` file